src/components/tiptap-node/paragraph-node/paragraph-node.scss

```
@@ -0,0 +1,267 @@
+    ::selection {
+      // background-color: var(--tt-selection-color);
+    }
+
+    .selection::selection {
+      background: transparent;
+    }
+  }
+
+  .selection {
+    display: inline;
+    // background-color: var(--tt-selection-color);
```

Stylelint (17.4.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)